x-pack/plugins/maps/public/connected_components/map/mb/tooltip_control/tooltip_popover.js

```diff
@@ -89,7 +90,7 @@ export class TooltipPopover extends Component {
       return null;
     }
 
-    const targetFeature = tooltipLayer.getFeatureById(featureId);
+    const targetFeature = tooltipLayer.getFeatureById(featureId, { mbProperties: {} });
```

Comment:
Including `mbProperties` here leaks implementation details of `TiledVectorLayer`. Will any other source ever use `mbProperties`. I think either `meta` should be passed into function or `getFeatureById` called with meta as null.

Reply:
see other comment